examples/waspello/package-lock.json

```diff
@@ -6,9 +6,9 @@
     "": {
```

Comment:
I got this error while trying to start the Wasp app with `wasp start`:
```
Error: Cannot find module @rollup/rollup-darwin-arm64. npm has a bug related to optional dependencies (https://github.com/npm/cli/issues/4828). Please try `npm i` again after removing both package-lock.json and node_modules directory.
```

I did what was suggested: `rm package-lock.json` and `rm -rf node_modules` and then ran `wasp start` again. The app worked, but I got a bit different `package-lock.json`. 

Is the app working for you with this `package-lock.json`? Could you maybe do the steps above and see what happens? (You could probably commit the new `package-lock.json` and then I can test again).

Reply:
Maybe we can pin point this to something, let me ask you: What are your `node` and `npm` versions? Which OS are you using? 

I'm on Node 20 and npm 10 on MacOS.

Reply:
I can reproduce the issue you describe on `main` branch as well. Can you double check? It might not be related to this PR.
<img width="1660" alt="Screenshot 2024-12-18 at 12 03 09" src="https://github.com/user-attachments/assets/104bb0d6-9d86-4ff4-8b70-54b0271a2267" />

Reply:
I created a PR to resolve the issue https://github.com/wasp-lang/wasp/pull/2423
Can you double check if it fixes it for your machine as well?

Reply:
Yep, it works great for me now 👍